classes/ImageManager.php

```diff
@@ -603,7 +603,7 @@ public static function cut($srcFile, $dstFile, $dstWidth = null, $dstHeight = nu
      * @param string $type
      * @param string $filename
      *
-     * @return false|resource
+     * @return false|resource|GdImage
```

Comment:
```suggestion
     * @return false|resource|\GdImage
```

@PrestaEdit Could you try it ?